src/components/swapv2/LimitOrder/const.ts

```diff
@@ -54,3 +54,9 @@ export const CLOSE_ORDER_OPTIONS = [
     value: LimitOrderStatus.EXPIRED,
   },
 ]
+
+export const NUMBERS = {
+  GAS_AMOUNT_ETHEREUM: 1_200_000,
+  ETH_USD_THRESHOLD: 300,
+  REST_USD_THRESHOLD: 10,
+}
```

Comment:
this config is different between chains, so I suggest change it to:
```
const _USD_THRESHOLD: {[chainId: number]: number} = {
  [ChainId.MAINNET]: 300
}
export const USD_THRESHOLD = new Proxy(_USD_THRESHOLD, {
  get(target, p) {
    const prop = p as any as ChainId
    if (p && target[prop]) return target[prop]
    return 10
  },
})
```

Reply:
this hardcode will remove soon 🌵 😿

Reply:
okay, so please note a // TODO(Danh): :pepe-laugh: